refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/test/java/de/muenchen/refarch/integration/dms/adapter/out/fabasoft/FabasoftAdapterTest.java

```diff
@@ -54,189 +55,197 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @WireMockTest()
+@SuppressWarnings({ "PMD.UnitTestShouldIncludeAssert", "PMD.CouplingBetweenObjects" })
```

Comment:
Why is `PMD.UnitTestShouldIncludeAssert` required here? Some methods have assertions in place. Please only add the suppression annotation where needed on individual methods.

Reply:
Don't really know why i suppressed it. Removed it.